apps/web/pages/api/teams.ts

```diff
@@ -1,6 +1,8 @@
 import type { NextApiRequest, NextApiResponse } from "next";
 
+import { closeComUpsertTeamUser } from "@calcom/lib/sync/SyncServiceManager";
 import prisma from "@calcom/prisma";
+import { MembershipRole } from "@calcom/prisma/client";
```

Comment:
@zomars Any idea why this import does not work in `pages`? it works in components and everywhere else I tried.

Reply:
It should be `import { MembershipRole } from "@prisma/client";`